src/DataMiner/SLManagedAutomation/UIBlockDefinition.cs

```diff
@@ -594,23 +594,19 @@ public class UIBlockDefinition
 		public int RowSpan { get; set; }
 
 		/// <summary>
-		/// Gets or sets the style of the dialog box.
+		/// Gets or sets the style of the dialog box item.
 		/// </summary>
-		/// <value>The style of the dialog box.</value>
+		/// <value>The style of the dialog box item.</value>
 		/// <remarks>
-		/// <para>It is possible to add one of three title styles to a text block:</para>
-		/// <list type="bullet">
-		/// <item><description>“Title1” (highest level, all lower case)</description></item>
-		/// <item><description>“Title2” (medium level, all upper case)</description></item>
-		/// <item><description>“Title3” (lowest level, all upper case)</description></item>
-		/// </list>
-		/// <para>If you want to set the style of a text block to regular text, leave the style property empty.</para>
+		/// <para>It is possible to set a style on some dialog box items.</para>
+		/// <para>The supported styles can be accessed through const strings on the Style class, subdivided per control type (Button, Text, ...).<br/>
+		/// The Style class is available from DataMiner 10.3.1 onwards. On older DataMiner versions you can use the StaticText styles 'Title1', 'Title2' and 'Title3'.</para>
```

Comment:
Also add the main release where the feature will be available in, same for the AppendButton method

Reply:
This is a new feature, so it will only be merged into RC, so no need to split it up into feature and main release. 
All DataMiner versions equal to or bigger than 10.3.1 will have this functionality.

Reply:
The main release version should be mentioned too, just for the sake of clarity. Otherwise, people on the main release track won't know whether it's available to them in 10.3.0, or if they need to wait for 10.4.0.

Reply:
Ok, added.